Formula/libeatmydata.rb

```diff
@@ -0,0 +1,37 @@
+class Libeatmydata < Formula
+  desc "Because fsync() should be a no-op"
+  homepage "https://www.flamingspork.com/projects/libeatmydata/"
+  url "https://github.com/stewartsmith/libeatmydata/releases/download/v130/libeatmydata-130.tar.gz"
+  sha256 "48731cd7e612ff73fd6339378fbbff38dd3bcf6c243593b0d9773ca0051541c0"
+  license "GPL-3.0-or-later"
+  head "https://github.com/stewartsmith/libeatmydata.git", branch: "master"
+
+  depends_on "autoconf"         => :build
+  depends_on "autoconf-archive" => :build
+  depends_on "automake"         => :build
+  depends_on "libtool"          => :build
+  depends_on "coreutils"        => :test
+
+  on_linux do
+    depends_on "strace" => :test
+  end
+
+  def install
+    system "autoreconf", "-vfi"
+    system "./configure", *std_configure_args,
+                          "--disable-option-checking",
+                          "--disable-silent-rules"
+    system "make", "install"
+  end
+
+  test do
+    ENV.prepend_path "PATH", Formula["coreutils"].opt_libexec/"gnubin"
+    system "#{bin}/eatmydata", "sync"
+    if OS.linux?
+      test_match = "grep -E '(^[a-z]*sync|O_SYNC| exited with 0 )'|wc -l"
+      # Verify via strace(1) that sync(1) just exits under eatmydata without sync(2) being called.
+      assert_match "1\n",
+        shell_output("#{bin}/eatmydata #{Formula["strace"].opt_bin}/strace sync 2>&1|#{test_match}")
+    end
```

Comment:
```suggestion
    return if OS.mac?

    output = shell_output("#{bin}/eatmydata strace sync 2>&1")
    refute_match(/^[a-z]*sync/, output)
    refute_match("O_SYNC", output)
    assert_match(" exited with 0 ", output)
```
This is supposed to be the idea of the test, no?

Reply:
Yes, that would work.

About using strace from `#{Formula["strace"].opt_bin}/strace`, the reason for that is because no all Linux distributions include strace by default, but by depending on the brew version it's surely available... is there a more proper way to express that?

Also, now that I think of it ... if you use SELinux, you can setsebool deny_ptrace which disables the use of ptrace, (which would break the test), and there are similar options for Yama, etc.... is that a concern here, or perhaps a comment should be added to note that?

Reply:
We don't really aim for the test to be portable; we just need it to work in CI. There, `HOMEBREW_PREFIX/bin` is first in `PATH` so you get Homebrew `strace`. We also don't use SELinux, so I wouldn't worry about it.

Reply:
Got it.  The last commit then isn't necessary, but it shouldn't hurt.  Otherwise, I think it's good now.